### PR TITLE
fixes #2493 Unable to load audit, underlying cause No such file to load -- audit

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -29,8 +29,9 @@ class ActiveSupport::TestCase
     # Note: You'll currently still have to declare fixtures explicitly in integration tests
     # -- they do not yet inherit this setting
 
+    set_fixture_class :hosts => Host::Base
+    #set_fixture_class :audits => Audited::Adapters::ActiveRecord
```
Comment (ohadlevy) [inline test/test_helper.rb]: why these changes?

File: lib/audit.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class Audit
```
Comment (ohadlevy) [inline lib/audit.rb]: why do we need to define the class?

Comment (isratrade) [inline lib/audit.rb]: this prevents the following error message when loading the 'audits' fixture along with a stack trace in the log/test.rb

http://pastie.org/7943990

It is ignored by rake test, but this error causes problems for spork-minitest.

Comment (ohadlevy) [inline lib/audit.rb]: but does it work correctly?  /me never tried to define two classes that one inherits and another does not... maybe we should restrict this only to test env?

Comment (isratrade) [inline lib/audit.rb]: I think re-opening class doesn't hurt anything.  I will restrict it to the test environment though.

**Comment by isratrade** [comment]
recommited.

File: lib/audit.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# this file prevents the error with loading fixtures
+# Unable to load audit, underlying cause No such file to load -- audit
+if Rails.env.test?
```
Comment (ohadlevy) [inline lib/audit.rb]: should we put it in lib then? maybe move it to /test?

Comment (isratrade) [inline lib/audit.rb]: yea, it works with the file in /test
however, it did not work by placing the class Audit; end; in test_helper.rb

**Comment by isratrade** [comment]
@domcleal, I think this PR  can be merged.

**Comment by ohadlevy** [comment]
merged at fa48699 thanks!
